components/PartnerCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const PartnerCard = ({ name, logoURL, url }) => {
+    return (
+        <div className="inline-block">
+            <a href={url} target="_blank">
+                <span
+                    className=" max-w-md shadow my-3 inline-block mx-6 border-2 border-gray-300
+        dark:border-gray-900 cursor-pointer hover:transform transition duration-300 ease-in-out hover:scale-105">
```

Comment:
Remove the Hover and add Small Border Radius

Reply:
Changes done.

Reply:
Merge it Then